packages/preferences-controller/package.json

```diff
@@ -63,7 +63,7 @@
     "typescript": "~5.2.2"
   },
   "peerDependencies": {
-    "@metamask/keyring-controller": "^17.0.0"
+    "@metamask/keyring-controller": "^18.0.0"
```

Comment:
This is a peer dependency update — can we include this in the release as well for similar reasons mentioned above?

Reply:
Major or minor?

Reply:
Major.